src/lib.rs

```diff
@@ -7,7 +7,7 @@
 //!
 //! ```no_run
 //! use winit::event_loop::EventLoop;
-//! let event_loop = EventLoop::new();
+//! let event_loop = EventLoop::new().unwrap();
```

Comment:
`?` in doctests are possible if you put `# Ok::<winit::error::EventLoopError>(())` at the end

Reply:
That's the same as with the rest, we'd have to change for everything at once.